dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/resources/OpenAPI.nlsprops

```diff
@@ -1,25 +1,26 @@
-#CMVCPATHNAME N/A
-#COMPONENTPREFIX CWMMC
-#COMPONENTNAMEFOR CWMMC MicroProfile OpenAPI API 
-#NLS_ENCODING=UNICODE
-#NLS_MESSAGEFORMAT_NONE
-#ISMESSAGEFILE true
-# #########################################################################
-###############################################################################
-# Copyright (c) 2009, 2011 IBM Corporation and others.
+# Copyright (c) 2018 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 #
 # Contributors:
 #     IBM Corporation - initial API and implementation
-###############################################################################
-# This file follows the WebSphere Message Guidelines.
-# For more information, visit: 
-# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
-#
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO MicroProfile OpenAPI API
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# Message prefix block: CWWKO1650 - CWWKO1699
+
+OPENAPI_DOCUMENT_VALIDATION_ERROR=CWWKO1650E: Validation of the OpenAPI document produced the following error. Message: {0}, Location: {1}
+OPENAPI_DOCUMENT_VALIDATION_ERROR.explanation=The OpenAPI document is invalid.
+OPENAPI_DOCUMENT_VALIDATION_ERROR.useraction=Provide a valid OpenAPI document that resolves the reported error.
 
-#-----------------------------------------------------------------------------------------------------------------------------
-# Messages used by OpenAPI Implementation
-#-----------------------------------------------------------------------------------------------------------------------------
\ No newline at end of file
+OPENAPI_DOCUMENT_VALIDATION_WARNING=CWWKO1651W: Validation of the OpenAPI document produced the following warning. Message: {0}, Location: {1}
+OPENAPI_DOCUMENT_VALIDATION_WARNING.explanation=The OpenAPI document contains data that is not in the recommended format.
```

Comment:
Is the fact that the format is recommended but not required the reason this is a warning rather than an error? Where can they go to find more information about the recommended format?

Reply:
Thanks for looking at the messages @cjcarpen 

OpenAPI Specification (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) describes the format of elements. We issue warnings when the document contains data that's not in the recommended format or it was unnecessary and would not have any effect and most likely be ignored by the consumers of the document.